docs/docs/train-tutorial.md

```diff
@@ -0,0 +1,338 @@
+# User Tutorial: Training a Model on Labaled Videos
```

Comment:
in this section, we should briefly mention the purpose of the `from_scratch` param, which can be used to instantiate the model from base weights rather than use our weights

Reply:
@ejm714 curious to get your opinion on this? should we surface `from_scratch` in the training tutorial or is this more of an advanced feature that we can leave in `configurations.md`?

Reply:
this should not be in the tutorials. it's unlikely anyone will have the quantity of videos we have where it would make sense to start from scratch